### ISSUE TITLE
Handle comments that look like HTML tags but aren't

### ISSUE THREAD

**Issue by coryan** [googleapis/google-cloud-rust#381]
Consider this:

https://github.com/googleapis/google-cloud-rust/blob/2c05b4823fbcf6364820907e2709a1502a39f89d/src/gax/src/error/rpc/generated/mod.rs#L199-L200

Rustdoc treats `<ip ...>` and `<Google ...>` as HTML tags.  This creates two problems:

1. rustdoc warngs complains that they are not closed, and
    ```
    warning: unclosed HTML tag `ip`
    --> src/gax/src/error/rpc/generated/mod.rs:198:9
        |
    198 | /         /// The subject on which the quota check failed.
    199 | |         /// For example, "clientip:<ip address of client>" or "project:<Google
    200 | |         /// developer project id>".
        | |___________________________________^
    ```
1. rustdoc removes them from the output:
    ![image](https://github.com/user-attachments/assets/bb325ca1-bcc4-40f3-bfbc-40feadce4831)

Either we change all the cases of placeholders using `<... >`, or we escape these placeholders in the Rust code to keep these in the documentation output.

To my knowledge, the only relevant API ( [aip/192](https://google.aip.dev/192) says nothing about such "almost looks like a tag" problems.

**Comment by coryan** [comment]
Maybe all we need to do is disable this warning for generated code:

https://doc.rust-lang.org/rustdoc/lints.html#invalid_html_tags

**Comment by coryan** [comment]
FWIW, in #409 I disabled the warnings about these things.  The generated documentation skips the thing that looks like a tag, so the problems from the description remains.
